test/expect-tests/dune_rpc/dune_rpc_tests.ml

```diff
@@ -54,14 +54,11 @@ let setup_client_server () =
   let connect () = Chan.connect client_chan server_chan in
   (client_chan, sessions, connect)
 
-let test ?(private_menu = []) ?no_real_methods ~client ~handler ~init () =
-  let () =
-    match no_real_methods with
-    | Some () -> ()
-    | None ->
-      Handler.implement_notification handler Procedures.Public.shutdown
-        (fun _ _ -> failwith "shutdown called")
-  in
+let test ?(private_menu = []) ?(no_real_methods = false) ~client ~handler ~init
+    () =
+  if not no_real_methods then
```

Comment:
Double negation is a bit silly here. Why not call it `has_real_methods`, get rid of the negation here and default to true.

Reply:
good point